lib/vsprintf.c

```diff
@@ -2236,6 +2240,10 @@ early_param("no_hash_pointers", no_hash_pointers_enable);
  *
  * Note: The default behaviour (unadorned %p) is to hash the address,
  * rendering it useful as a unique identifier.
+ *
+ * There is also a `%pA` format specifier, but it is only intended to be used
+ * from Rust code to format `core::fmt::Arguments`. Do *not* use it from C.
+ * See rust/kernel/print.rs for details.
```

Comment:
Since the surrounding C code does not use Markdown-like syntax, please avoid it there:

```suggestion
 * There is also a %pA format specifier, but it is only intended to be used
 * from Rust code to format core::fmt::Arguments. Do *not* use it from C.
 * See rust/kernel/print.rs for details.
```